textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTestConstants.java

```diff
@@ -104,4 +104,58 @@
 
     
     
+    public static final String NEWS_NUMBER = "news_number";
+    public static final String NEWS_TITLE = "news_title";
+    public static final String NEWS_BODY = "news_body";
+    
+    public static final Attribute NEWS_NUM_ATTR = new Attribute(NEWS_NUMBER, FieldType.INTEGER);
+    public static final Attribute NEWS_TITLE_ATTR = new Attribute(NEWS_TITLE, FieldType.TEXT);
+    public static final Attribute NEWS_BODY_ATTR = new Attribute(NEWS_BODY, FieldType.TEXT);
+    
+    public static final Schema NEWS_SCHEMA = new Schema(NEWS_NUM_ATTR, NEWS_TITLE_ATTR, NEWS_BODY_ATTR);
+    
+    public static List<ITuple> getNewsTuples() {
+        return Arrays.asList(
+                new DataTuple(NEWS_SCHEMA, 
+                        new IntegerField(1),
+                        new TextField("UCI marchers protest as Trump begins his presidency"),
```

Comment:
In the future, it will be good to avoid political news articles and/or uci-related articles to avoid unnecessary contraversary.  We want to keep this project "neutral."